develop-docs/sdk/telemetry/profiles/sample-format-v2.mdx

```diff
@@ -0,0 +1,303 @@
+- `slow_frame_renders`: when a slow frame was detected.
+
+A measurement value can contain the following attributes:
+- `elapsed_since_start_ns`: number of nanoseconds since the start of the profile
+  (stored in `timestamp`). It should be wall time.
+- `value`: value for the measurement, as a `float64`. We accept `float64`
+  formatted as `string` as well.
+
+These are the values accepted for `unit`:
+- `nanosecond`
+- `ns`
+- `hertz`
+- `hz`
+- `byte`
+- `percent`
+
+```json
+{
+  "cpu_0": {
+    "unit": "percent",
+    "values": {
+      "elapsed_since_start_ns: "1234567890",
+      "value": 12.64,
+    }
+  }
+}
+```
+
+`platform`
+
+: *String, required.* This value represents the platform the SDK is submitting
+from.
+
+`release`
+
+: *String, required.* The release version of the application.
+
+**Release versions must be unique across all projects in your organization**.
+This value can be the git SHA for the given project, or a product identifier
+with a semantic version (suggested format `my-project-name@1.0.0`).
+
+`environment`
+
+: *String, recommended.* The environment name, such as `production` or
+`staging`. The default value should be `production`.
+
+`version`
+
+: *String, required.* Represents the version of the Sample format.
+
+Acceptable values are:
+- `"2"`
+
+`profile`
+
+: *Object, required.* It contains all the raw data collected when profiling. See
+[profile data](#profile-data).
+
+
+### Profile data
+
+The profile data consists of
+- sample, which specifies the start time, their corresponding thread and a stack,
+- stacks, which are lists of frames,
+- frames, which identify the function and/or line of code for each item in a stacktrace,
+- thread metadata, with info about threads.
+
+The selected format is trying to keep the amount of data we transfer to the server as limited as possible.
+
+```json
+{
+  "profile": {
+    "samples": [
+      { ... },
+      ...
+    ],
+    "stacks": [
+      [ ... ],
+      ...
+    ],
+    "frames": [
+      { ... },
+      ...
+    ],
+    "thread_metadata": {
+      "259": {
+        ...
+      }
+    }
+  }
+}
+```
+
+When collecting a stack trace, you will:
+1. Capture the stack trace.
+2. Capture an absolute timestamp of when the stack trace was "captured".
+3. Capture the thread ID where this stack trace was captured.
+4. Create a list of indices for each frames (add it to the frame list if needed
+or use the existing frame's index).
+5. Check if that stack exists in the list of stacks and if it doesn't, add it to the list of stacks
+6. Create a sample containing the thread ID, the timestamp and the
+stack ID.
+
+You should collect samples at a frequency of 101Hz, or roughly once every 10
+milliseconds.
+
+`samples`
+
+: *List, required* Contains a list of sample object captured during execution.
+Each sample can contain the following attributes:
+
+- `timestamp`: *64-bit floating point, required.* Contains the Unix timestamp 
```

Comment:
We'd like microsecond precisions instead of millisecond. In general, the more precise you are the better.